### PR TITLE
Provide link and description of the program

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
ping @retokromer @JeromeMartinez @kieranjol

File: notimetowait2.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ REGISTRATION IS NOW OPEN
 
 [http://bit.ly/nttw2signup](http://bit.ly/nttw2signup)
 
-## Call for Proposals:
+## Program:
 
-CALL FOR PROPOSALS IS OPEN UNTIL SEPTEMBER 22, 2017.
+Below is the preliminary program.  Please note that sessions are still being finalized and program times, titles, and speakers are subject to change. The program will be updated again by October 5th. Questions or comments about the program may be set to [info@mediaarea.net](mailto:info@mediaarea.net) or provided as a comment within the Google spreadsheet.
```
Comment (JeromeMartinez) [inline notimetowait2.md]: (double space after first period)

I would remove "The program will be updated again by October 5th." as this is already said "preliminary", and program may change also after Oct 5.

**Review by retokromer — APPROVED** [review]
There is a double space after the period «preliminary program.  Please note»

File: notimetowait2.md (hunk 1 above)
Comment (retokromer) [reply notimetowait2.md]: +1

**Comment by dericed** [comment]
Thanks @retokromer @JeromeMartinez, updated.

**Review by JeromeMartinez — APPROVED** [review]

**Review by ablwr — APPROVED** [review]

**Review by retokromer — APPROVED** [review]

File: notimetowait2.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ REGISTRATION IS NOW OPEN
 
 [http://bit.ly/nttw2signup](http://bit.ly/nttw2signup)
 
-## Call for Proposals:
+## Program:
 
-CALL FOR PROPOSALS IS OPEN UNTIL SEPTEMBER 22, 2017.
+Below is the preliminary program. Please note that sessions are still being finalized and program times, titles, and speakers are subject to change. Questions or comments about the program may be set to [info@mediaarea.net](mailto:info@mediaarea.net) or provided as a comment within the Google spreadsheet.
```
Comment (kieranjol) [inline notimetowait2.md]: Should this be 'sent to'? Otherwise, LGTM.

Comment (dericed) [reply notimetowait2.md]: fixed

**Pull request merged**